### DISCUSSION TITLE
Zeek wont start sudo so-zeek-start gives following error

### DISCUSSION THREAD

**Discussion by wege2003** [Security-Onion-Solutions/securityonion#3610] [Unsupported Versions] [answered]
Hi there i get following error wenn starting the zeek docker (Screenshot)
How can i remove this error?

I used the Cent OS Security Onion ISO.

Greets LaviS

![SO-Zeek](https://user-images.githubusercontent.com/31622452/112282278-3d309600-8c87-11eb-8d66-07adfbb99350.png)

**Comment by weslambert** [comment, marked as answer]
This looks similar to your other question.  What kind of node are you trying to run this from? Normally this happens when you are trying to start Zeek on a node that is not supposed to be running Zeek.
